tests/30rooms/06invite.pl

```diff
@@ -87,6 +87,9 @@ sub inviteonly_room_fixture
          $content->{membership} eq "invite" or
             die "Expected membership to be 'invite'";
 
+         require_json_keys( $event, qw( invite_room_state ));
```

Comment:
Ah. I can make this one optional then, but then what to do of the later test below? I guess it should just stop early if it didn't find the result.